AdyenCard/Components/Card/CardViewController.swift

```diff
@@ -244,6 +244,16 @@ internal class CardViewController: FormViewController {
 
         items.triggerInfoEvent(of: .selected, target: .dualBrandButton, brands: [selectedBrand])
     }
+
+    private func showCoBadgedCardsUI(for brands: [CardBrand]) {
+        let allowedCoBadgedCardTypes = ["cartebancaire", "bcmc", "dankort"]
```

Comment:
Let's do this with the CardType enum not rawvalue.  Can also define this as a constant either on cardComponent to be injected here or can define at top of this class. `let allowedCoBadgedCardTypes: [CardType] = [.bcmc, .....]`

Lastly let's confirm these on our integration tests

Reply:
Thanks for this suggestion :) de173ad7f48cb72548e6293fd0f38937592d1781